### PR TITLE
Expanded README.md explanation of errfcb plugin

### PR DESCRIPTION
**Pull request by cxw42**
I have recently discovered `asynquence` and am greatly enjoying it --- thank you!  This is my first attempt to contribute; feedback is welcome.

The `errfcb()` contrib plugin is very useful to me, since the resulting callback can be adapted for use by Chrome extension API functions ([my use case](https://github.com/cxw42/TabFern)).  However, it took me a bit of head-scratching to understand it.  This PR expands the documentation of that plugin for the sake of relative newbies like me.

Thank you for considering this request!

### CODE REVIEW

File: contrib/README.md [hunk 1]
```diff
@@ -294,9 +294,11 @@ As long as either the native `Promise` is there, or that global has been spec-co
 
 ### `errfcb` Plugin
 
-`errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.
+The `errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.  Calling `errfcb()` also stops the sequence from proceeding until the "error-first" callback is invoked.
```
Comment (getify) [inline contrib/README.md]: I see what you're trying to do with this added sentence, and why that helps with clarity. But I'd actually recommend this paragraph be edited for even more clarity, so it reads like this:

----

The `errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` creates a step in the sequence that will wait to proceed, and returns an "error-first" style (aka "node-style") callback -- suitable for any utility that expects such a callback -- to signal this waiting sequence step.

File: contrib/README.md [hunk 2]
```diff
@@ -294,9 +294,11 @@ As long as either the native `Promise` is there, or that global has been spec-co
...
 
-If the "error-first" callback is then invoked with the first ("error") parameter set, the main sequence is flagged for error as usual. Otherwise, the main sequence proceeds as success. Messages sent to the callback are passed through to the main sequence as success/error as expected.
+If the "error-first" callback is then invoked with the first ("error") parameter having a truthy value, the main sequence is flagged for error as usual.  The value of the "error" parameter is provided to `.or()` callbacks as the failure reason.
+
+If the "error" parameter has a falsy value, the main sequence proceeds as success.  Any other values provided to the callback are passed through to the main sequence and can be accessed by later steps.
```
Comment (getify) [inline contrib/README.md]: Let's reword this just a bit for more clarity:

----

Any other values provided to the callback are passed through as normal messages to the main sequence, and can be accessed by the next step in the sequence.

File: contrib/README.md (hunk 1 above)
Comment (cxw42) [reply contrib/README.md]: Done, and thanks for the quick response!  I pulled the "suitable for" out to a separate sentence just to break it up a bit.

**Comment by getify** [comment]
Thanks!

**Pull request merged**